Reading List/Reading List/Views/BookTableViewCell.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  BookTableViewCell.swift
+//  Reading List
+//
+//  Created by Angel Buenrostro on 1/18/19.
+//  Copyright © 2019 Lambda School. All rights reserved.
+//
+
+import UIKit
+
+class BookTableViewCell: UITableViewCell, BookTableViewCellDelegate {
```

Comment:
No need to conform `BookTableViewCellDelegate` protocol
```suggestion
class BookTableViewCell: UITableViewCell {
```